saleor/graphql/shipping/types.py

```diff
@@ -135,6 +153,10 @@ def resolve_maximum_order_weight(
     ):
         return convert_weight_to_default_weight_unit(root.node.maximum_order_weight)
 
+    @staticmethod
+    def resolve_zip_codes(root: ChannelContext[models.ShippingMethod], *_args):
+        return root.node.zip_codes.all()
```

Comment:
Here we should add a dataloader e.g. `ZipCodesByShippingMethodIdLoader`. It would be used in this way:
```
return ZipCodesByShippingMethodIdLoader(info.context).load(root.node.id)
```